wormhole-connect/src/views/v2/Bridge/AssetPicker/ChainList.tsx

```diff
@@ -136,11 +148,11 @@ const ChainList = (props: Props) => {
             ?.toLowerCase()
             .includes(chainSearchQuery.toLowerCase());
         })
-      : topChains;
+      : props.chainList;
 
     return (
-      <List>
-        <ListItem>
+      <List className={classes.searchList}>
+        <ListItem className={classes.searchBar}>
```

Comment:
@nikarm22 Can we leave the search bar outside the scrollable section? The scrollbar should only appear for the list of chains. I've previously added the search bar as part of the list for better alignment, but for this we can move it out.